model/host/host.go

```diff
@@ -1029,3 +1029,33 @@ func InsertMany(hosts []Host) error {
 	return errors.WithStack(db.InsertMany(Collection, docs...))
 
 }
+
+// CountContainersRunningAtTime counts how many containers were running on the
+// given parent host at the specified time, using the host StartTime and
+// TerminationTime fields.
+func (h *Host) CountContainersRunningAtTime(timestamp time.Time) (int, error) {
```

Comment:
Is this query indexed?

Reply:
ParentID is indexed, but StartTime and TerminationTime are not. I think this should be fine? I asked John and he said that ParentID has high enough cardinality. The query won't have to compare that many start and end times for a given ID. Let me know if you think otherwise.

Reply:
sgtm